source/blog/2017-03-02-announcing-hanami-100beta2.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: Announcing Hanami v1.0.0.beta2
+date: 2017-03-02 13:33 UTC
+tags: announcements
+author: Luca Guidi & Oana Sipos
+image: true
+excerpt: >
+  Small fixes preparing the stable 1.0.0 version
+---
+_From now on, **Hanami API's are stable and won't be changed until 2.0**._
```

Comment:
Maybe it will work better indeed. Thinking twice, not such a fan of italic anymore :zipper_mouth_face: